admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,199 @@
+
+Pod distribution among nodes:
+small-pod
+	- 192.168.124.214: 26 instance(s)
+	- 192.168.124.120: 26 instance(s)
+```
+
+[admin-guide-cluster-capacity-job]
+### Running cluster capacity as a Job
+
+Below is an example of a Job spec (`cluster-capacity-job.yaml`) that runs cluster capacity image as a job in a pod.
+Note that input pod spec (`pod.yaml`) to cluster capacity analysis is mounted in a volume using a `ConfigMap`.
+
+```yaml
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cluster-capacity-job
+spec:
+  parallelism: 1
+  completions: 1
+  template:
+    metadata:
+      name: cluster-capacity-pod
+    spec:
+        containers:
+        - name: cluster-capacity
+          image: openshift/origin-cluster-capacity
+          imagePullPolicy: "Never"
+          volumeMounts:
+          - mountPath: /test-pod
+            name: test-volume
+          env:
+          - name: CC_INCLUSTER
+            value: "true"
+          command:
+          - "/bin/sh"
+          - "-ec"
+          - |
+            /bin/cluster-capacity --podspec=/test-pod/pod.yaml --verbose
+        restartPolicy: "Never"
+        serviceAccountName: cluster-capacity-sa
+        volumes:
+        - name: test-volume
+          configMap:
+            name: cluster-capacity-configmap
+```
+
+Also note that the above pod spec specifies the following environment variable which is required to
+let the cluster capacity command know that it is running inside a cluster as a pod.
+
+```yaml
+env:
+- name: CC_INCLUSTER
+  value: "true"
+```
+
+By default, this pod would be created in the `default` namespace by using `default` service account. However, the above
+pod spec refers to a different service account (`cluster-capacity-sa`) in the `default` namespace created as below:
+
+```
+$ oc create sa cluster-capacity-sa
+```
+
+This service account `cluster-capacity-sa` is assigned a new cluster scope role `cluster-capacity-role` to be able to access
+resources required to run cluster capacity command inside a pod. The role `cluster-capacity-role` is created as:
+
+```shell
+$ cat << EOF| oc create -f -
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: cluster-capacity-role
+rules:
+- apiGroups: [""]
+  resources: ["pods", "nodes", "persistentvolumeclaims", "persistentvolumes", "services"]
+  verbs: ["get", "watch", "list"
+EOF
+```
+
+The next step binds the service account `cluster-capacity-sa` to the new cluster role `cluster-capacity-role`:
+
+```shell
+$ oadm policy add-cluster-role-to-user cluster-capacity-role system:serviceaccount:default:cluster-capacity-sa
+```
+
+Please note that in the above Job spec, we used a `ConfigMap` `cluster-capacity-configmap` to mount input
+pod spec file `pod.yaml` into a volume `test-volume` at the path `/test-pod`. The ConfigMap can be created as:
+
+```shell
+$ oc create configmap cluster-capacity-configmap --from-file=pod.yaml=pod.yaml
```

Comment:
i think we should reverse the order of this doc.

1. create the cluster role
2. create the service account
3. add the role to service account
4. define your pod spec
5. make it available in config map
6. run the job (now that you have done what is needed)